app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,6 @@
 class PostsController < ApplicationController
+  before_action :authenticate_user!, only: %i[create destroy]
+
```

Comment:
- [ ] Kindly use `CanCanCan` for Post authorization in the `Post Controller`

```ruby
  class PostsController < ApplicationController
    load_and_authorize_resource

    ...
  end
```

Reply:
Thank you for the feedback implemented the required change